docs/src/guide/design-tokens.md

```
@@ -10,15 +12,24 @@ That's why vuecs has both: themes pick the class names, tokens pick the colors.
 
 ## The token chain (top to bottom)
 
+When `@vuecs/theme-tailwind` is loaded:
+
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)